app/pages/analyst/applications.tsx

```diff
@@ -50,22 +50,26 @@ class Applications extends Component<Props> {
     return {
       variables: {
         orderByField: 'operator_name',
-        direction: 'ASC'
+        direction: 'ASC',
+        max_results: 10,
```

Comment:
I think that we can set this to 20

Reply:
I was thinking we could make this something the user could control, like you see sometimes see something like: `results **10** 20 50` but maybe that's an enhancement for later?

Reply:
Haha, every time I see one of these I set it to the maximum so I don't have to deal with too much pagination - at the expense of server work. I'd say if we leave it at 10 or 20 it's fine for our user's purposes?